docs/6.x/installation.md

```diff
@@ -1,57 +1,56 @@
+Instance from a Cluster Image.  Just like a virtual machine (VM) image or AWS AMI 
+can be used to create machine instances, Gravity Cluster Images can 
+be used to create Cluster Instances.
 
 #### Deployment Methods
 
-Gravity supports two methods to create clusters from cluster images:
+Gravity supports two methods to create Clusters from Cluster Images:
 
-* Command Line (CLI) method, suitable for scripting.
-* Graphical installer which serves a web UI, assisting users in cluster
-  creation.
+* The command line interface (CLI) installer which is suitable for scripting.
+* The graphical user interface (GUI) installer which serves an installation wizard
+in a web browser, assisting users in Cluster creation.
 
-Both installation methods allow users to create a new Kubernetes cluster from a
-cluster image on arbitrary Linux hosts. Because a cluster image has no external
-dependencies, both installation methods will work even in air-gapped server
-rooms.
+Both installation methods allow users to create a new Cluster from a
+Cluster Image on Linux hosts. Because a Cluster Image has no external
+dependencies, both installation methods will work behind firewalls or even in 
+air-gapped server rooms.
 
 #### Prerequisites
 
-Every cluster image created with Gravity contains everything you need to 
-create a production-ready cluster, but there are still some pre-requisites
+Every Cluster Image created with Gravity contains everything you need to 
+create a production-ready Cluster but there are still some pre-requisites
 to be met:
 
-* You should have a valid cluster image, i.e. a `.tar` file. See [building images](/pack/) 
-  chapter for information on how to build one.
+* You need a valid Cluster Image, i.e. a `.tar` file. See the 
+  [Building Cluster Images](/pack/) section for information on how to build one.
 * One or more Linux hosts with a compatible kernel. They can be bare metal hosts,
   compute instances on any cloud provider, virtual machines on a private cloud, 
-  etc. Consult with [system requirements](/requirements/) to determine if your Linux hosts
-  are compatible with Gravity.
-* The hosts should be clean, i.e. they shouldn't contain any container
-  orchestrator on them, or even Docker.
-* The hosts should be able to connect to each other, i.e. to be on the same
+  etc. Consult with the [System Requirements](/requirements/) to determine if your 
+  Linux hosts are compatible with Gravity.
+* The hosts should be clean without Docker or any container
+  orchestrator running on them.
+* The hosts should be able to connect to each other, i.e. on the same
   private network.
-* You should have the ability to create DNS entries for public access points.
-* You you should be able to obtain a valid SSL/TLS certificates for HTTPS.
-
+* You need the ability to create DNS entries for public access points.
+* You need to be able to obtain a valid SSL/TLS certificates for HTTPS.
```

Comment:
Should be either `obtain valid SSL/TLS certificates` or `obtain a valid SSL/TLS certificate`